src/formats/tiff/types.zig

```diff
@@ -26,7 +26,37 @@ pub const ResolutionUnit = enum(u16) {
     cm = 3,
 };
 
-pub const TagId = enum(u16) { new_subfile_type = 254, image_width = 256, image_height = 257, bits_per_sample = 258, compression = 259, photometric_interpretation = 262, fill_order = 266, strip_offsets = 273, orientation = 274, samples_per_pixel = 277, rows_per_strip = 278, strip_byte_counts = 279, x_resolution = 282, y_resolution = 283, planar_configuration = 284, resolution_unit = 296, software = 305, color_map = 320, extra_samples = 338, sample_format = 339, unknown_1 = 700, unknown_2 = 34665, unknown_3 = 34675 };
+// zig fmt: off
```

Comment:
You don't need zig fmt off here, just put a comma for the last entry and zig fmt will format the enum exactly like that